sys/linux/sys.txt

```diff
@@ -78,6 +79,11 @@ type signalnoptr intptr[0:65]
 
 syz_execute_func(text ptr[in, text[target]])
 
+openat$sysfs(fd const[AT_FDCWD], dir ptr[in, glob["/sys/**/*"]], flags flags[open_flags], mode flags[open_mode]) fd_sys
+write$sysfs(fd fd_sys, buf buffer[in], count len[buf])
```

Comment:
These write/read/close don't do anything special beyond what the standard write/read/close do on fd's. And there also mmap/ioctl/poll/etc that can be executed on fd_sys.
If we don't provide more detailed operation (and I think it's not possible for _all_ files in the dir), it's better to do just:
```
openat$sysfs(fd const[AT_FDCWD], dir ptr[in, glob["/sys/**/*"]], flags flags[open_flags], mode flags[open_mode]) fd
```

Or, what's commonly written to these files? Maybe it's common to write integer values? If so, we could provide only write specialization that writes integers.